lib/trino-parquet/src/main/java/io/trino/parquet/reader/ParquetReader.java

```diff
@@ -387,7 +387,7 @@ private ColumnChunk readPrimitive(PrimitiveField field)
             // update batch size
```

Comment:
Commit message too long.
Please also add explanation what regression this fixes

Reply:
Please update the commit title to avoid truncation on GitHub and add an explanation in the commit message